sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/AuthenticationEventsTriggerAttribute.cs

```diff
@@ -19,14 +19,19 @@ public AuthenticationEventsTriggerAttribute()
         {
         }
 
-        /// <summary>Gets or sets the tenant identifier.</summary>
-        /// <value>The tenant identifier.</value>
-        public string TenantId { get; set; }
```

Comment:
Is this a breaking change?

Reply:
Yes sir. but we are okay with it. It will help us align with other MSFT auth libraries and it more correct than what we have had before.